### ISSUE TITLE
[JMAR-92] Object detection not working on top of App Conductor

### ISSUE THREAD

**Issue by jasPreMar** [jasPreMar/hyper-pointer#152]
**Linear:** https://linear.app/jason-marsh/issue/JMAR-92/object-detection-not-working-on-top-of-app-conductor
**Status:** Canceled

---
Object detection is failing to detect elements when hovering over the App Conductor. Need to investigate why accessibility/object detection breaks in this context.
